Please [let me know](mailto:ryan@gutenkunst.org) if you've published a paper using ∂a∂i.

## Primary ∂a∂i reference ##

  * RN Gutenkunst et al. "Inferring the joint demographic history of multiple populations from multidimensional SNP data" [PLoS Genetics 5:e1000695 (2009)](http://dx.doi.org/10.1371/journal.pgen.1000695); [arXiv:0909.0925](http://arxiv.org/abs/0909.0925)

> This paper introduces the method, and applies it to model human expansion out of Africa and the settlement of the New World. Additionally, the resulting Out of Africa demographic model was combined with a previously estimated distribution of selective effects to accurately predict the distribution of nonsynonymous mutations between populations. The 5 Mb of non-coding resequencing data used for demographic inference came from the Environmental Genome Project.

## Known publications using ∂a∂i ##

  * AM Andrés et al. "Targets of balancing selection in the human genome" [Molecular Biology and Evolution 26:2755 (2009)](http://dx.doi.org/10.1093/molbev/msp190) <br> R Nielsen et al. "Darwinian and demographic forces affecting human protein coding genes" <a href='http://dx.doi.org/10.1101/gr.088336.108'>Genome Research 19:838 (2009)</a></li></ul>

<blockquote>These papers describe selection scans in the human genome using resequencing data from 13,400 protein coding genes from 20 European-American and 19 African-American individuals. In both cases, they assess the significance of their extreme genes by comparing with the variation expected during neutral evolution, including demography. ∂a∂i was used after another demography inference method yielded controversial admixture proportion estimates for the African-American individuals. Interestingly, ∂a∂i estimated an average of 20% European admixture in the African-American individuals, without using any African data. This estimate of 20% is quite close to the typically accepted value.

  * X Yi et al. "Sequencing of 50 human exomes reveals adaptation to high altitude" [Science 329:75 (2010)](http://dx.doi.org/10.1126/science.1190371)

> This paper describes a scan for genes undergoing positive selection between Tibetans and Han Chinese. Statistical significance of outlier genes was determined by comparison with coalescent simulations based on a demographic model for the two populations inferred using ∂a∂i.

  * C Murray et al. "Cattle demographic history modelled from autosomal sequence variation" [Philosophical Transactions of the Royal Society B 365:2531 (2010)](http://dx.doi.org/10.1098/rstb.2010.0103)

> This paper describe demographic modeling of domestic and wild cattle from Africa, Europe, and India. Interestingly, the model suggests substantial admixture from South Asian aurochs.

  * J Xing et al. "Genetic diversity in India and the inference of Eurasian population expansion" [Genome Biology 11:R113 (2010)](http://dx.doi.org/10.1186/gb-2010-11-11-r113)

> This paper analyzes sequence from several Indian populations. ∂a∂i was used to infer demographic models connecting these populations with other continental groups (Europeans, Africans, and East Asians).

  * DP Locke et al. "Comparative and demographic analysis of orang-utan genomes" [Nature 469:529 (2011)](http://dx.doi.org/10.1038/nature09687)

> ∂a∂i was used to infer demographic models for Bornean and Sumatran orangutans, finding a more recent divergence than previously suspected and a larger effective population size in Sumatran than Bornean orangutans.

> ![http://dadi.googlecode.com/svn/wiki/OrangNatureCover.jpg](http://dadi.googlecode.com/svn/wiki/OrangNatureCover.jpg)

  * CE Ellison et al. "Population genomics and local adaptation in wild isolates of a model microbial eukaryote" [Proceedings of the National Academy of Sciences USA 108:2831 (2011)](http://dx.doi.org/10.1073/pnas.1014971108)

> ∂a∂i was used to infer a demographic model for Caribbean and Louisiana populations of fungus Neurospora. This model was used to test the significance of outliers in a scan for regions of the genome under selection.

  * J Molina et al. "Molecular evidence for a single evolutionary origin of domesticated rice" [Proceedings of the National Academy of Sciences USA 108:8351 (2011)](http://www.pnas.org/content/108/20/8351.abstract)

> ∂a∂i was used to infer and compare models for the domestication of indica and japonica rice from its wild progenitor. Surprisingly, the data were consistently better fit by a model with a single domestication event, rather than two. This result was then corroborated by analysis with BEAST.

  * JD Jensen and D Bachtrog "Characterizing the influence of effective population size on the rate of adaptation: Gillespie's Darwin Domain" [Genome Biology and Evolution (2011)](http://dx.doi.org)

> ∂a∂i was used to infer the demographic histories of Drosophila miranda and D. pseudoobscura, as a control when evaluating the prevalence of adaptive evolution versus genetic drift in molecular evolution.